docker/build.sh

```diff
@@ -4,11 +4,6 @@ set -eo pipefail
 
 IMAGE=${TARGET_IMAGE:-hivemq/hivemq-ce:snapshot}
```

Comment:
This whole script file is not used anymore.

Reply:
It is documented in the readme, that you can use this script to create a docker image locally.